test/tools/libtesteth/ImportTest.cpp

```diff
@@ -307,7 +343,7 @@ bytes ImportTest::executeTest(bool _isFilling)
             if (Options::get().verbosity > 2)
             {
                 std::cout << "Executed transaction: \n";
```

Comment:
Should we use boost loggers here? Or can this code be called before boost loggers are initialized?

Reply:
I think we should, however testeth has its own hacking around logging and handling verbosities, so I'm not going to touch it here.

Reply:
Makes sense, let's leave it alone